release_management/create_artifacts.py

```diff
@@ -55,13 +55,23 @@ def main():
 
     stratisd_parser.set_defaults(func=_stratisd_artifacts)
     stratisd_parser.add_argument("version", action="store", help="version")
+    stratisd_parser.add_argument(
```

Comment:
Add argument as a global argument, since it works the same for stratisd as for stratis-cli, similar to ```output_dir``` command-line argument.

Reply:
I had to adjust the shell script, but it works.